R/mod_metricGrid.R

```diff
@@ -37,7 +37,8 @@ metricGridServer <- function(id, metrics) {
     
       fluidRow(style = "padding-right: 10px", class = "card-group",
                map(column_vector_grid_split, 
-                   ~ column(width= 4,map(.x,~ metricBoxUI(session$ns(metrics()$name[.x])))))
+                   ~ column(width= 4,map(.x,~ metricBoxUI(session$ns(metrics()$name[.x]))))),
+      tags$em("* Provided for additional context. Not a {riskmetric} assessment, so this measure will not impact the risk score.")
```

Comment:
Hi @Robert-Krajcik, I read you and Jeff's conversation. I think it would make sense to check if their exists any `*` first before displaying this phrase.

Reply:
Hi @AARON-CLARK 
I think I have this resolved.
The message will appear in communityUsageMetrics
and in maintenanceMetics **if** any cards are non-metric cards